src/components/MultiHopTrade/components/MultiHopTradeConfirm/components/Hops.tsx

```diff
@@ -33,6 +33,8 @@ export const Hops = memo((props: HopsProps) => {
 
   if (!activeQuote || !firstHop || !swapperName) return null
 
+  const tradeId = activeQuote.id
```

Comment:
q: do we want to add a `selectActiveQuoteId` selector similar to the newly added `selectConfirmedQuoteTradeId` and have `<Hop />` select it internally vs. passing it down? 

Or we could possibly not select it in `<Hop />` altogether (still needs to be passed down a few-levels) and have the downmost consumer (`selectHopExecutionMetadata`) `selectActiveQuoteId` instead

Reply:
@0xApotheosis keeping this one open re: q, though the second part has been discussed and won't be actionned, see https://github.com/shapeshift/web/pull/7551#issuecomment-2290034322

Reply:
In this case I don't think we want to add `selectActiveQuoteId` as we already have an active quote here to pull the `id` from (which needs to be selected any way).